packages/office-ui-fabric-react/src/components/Button/Button.scss

```diff
@@ -351,7 +351,7 @@ $button-commandButtonHeight: 40px;
   background-color: transparent;
   color: $ms-color-neutralSecondary;
   padding: 0;
-  min-width: auto;
+  min-width: 0;
```

Comment:
When we have variants of a button (like an icon-only button) we implement that using a modifier class: `ms-Button--icon`. This overrides the styles of the default variant: `ms-Button`. This means we have to override values that would otherwise be fine as the browser defaults.

An alternative would be to add an explicit "default" modifier to every component so that we'd have `ms-Button--default`. Only styles that are shared amongst all of the variants would then be included in `ms-Button`. It's a good option to discuss, but I'd say it's out of scope for this issue.